### DISCUSSION TITLE
how to add async on dependency?

### DISCUSSION THREAD

**Discussion by anajgroup** [dnnsoftware/Dnn.Platform#5043] [Q&A]
hello every one

how can i add async attribute on default.aspx? 
`<dnncrm:ClientResourcePath Name="SkinPath" Path="<%# CurrentSkinPath %>"/>`

html attribute not works!

**Comment by bdukes** [comment]
Does [the _Additional Attributes_ section of the _Client Resource_ page in the docs](https://docs.dnncommunity.org/content/tutorials/client-resources/index.html#additional-attributes) help?

**Comment by anajgroup** [comment]
no thats not helpfull for dependencyhandlers.axd or webresource.axd. 
problem is about dnn core css and js files!

**Comment by ufoloko** [comment]
I think this function is very helpfull, mainly when you need to improve site performance score.
